README.rst

```diff
@@ -59,6 +59,26 @@ This will print:
 You get both IPv4 and IPv6 addresses. The later complete with
 flowinfo and scope_id.
 
+If you wish to include network interfaces that do not have a configured IP
+addresss, pass the `include_unconfigured` parameter to `get_adapters()`.
+Adapters with no configured IP addresses will have an zero-length `ips`
+property.  For example:
+
+.. code-block:: python
+
+    import ifaddr
+
+    adapters = ifaddr.get_adapters(include_unconfigured=True)
+
+    for adapter in adapters:
+        print "IPs of network adapter " + adapter.nice_name
+        if adapter.ips:
+            for ip in adapter.ips:
+                print "   %s/%s" % (ip.ip, ip.network_prefix)
```

Comment:
Oh, and also can you add parentheses to those print calls for Python 3 compatibility?

Reply:
👍